build/pythontool.py

```diff
@@ -1,10 +1,15 @@
 from waflib import Options
-import os, re
+import os, sys, re
 
 def options(opt):
     opt.load('python')
-    opt.add_option('--disable-python', action='store_false', dest='python',
-                   help='Disable python', default=True)
+    if sys.platform == 'sunos5':
+        opt.add_option('--enable-python', action='store_true',
+                dest='python', help='Disable python', default=False)
```

Comment:
Add a comment for why we're doing this... something like "The Solaris Studio compiler cannot compile some of the C++ bindings that Swig produces so default to no Python bindings on Solaris so the build doesn't fail"